obspy/io/ah/core.py

```diff
@@ -228,6 +227,219 @@ def _unpack_trace(data):
         return st
 
 
+def _write_ah1(stream, filename):
+    """
+    Writes a Stream object to an AH v1 waveform file.
+
+    :type stream:
+    :param stream: The ObsPy Stream object to write.
+    :type filename: str
+    :param filename: open file, or file-like object
+
+    """
+    packer = xdrlib.Packer()
+
+    for tr in stream:
+        if hasattr(tr.stats, 'ah'):
+            packer = _pack_trace_with_ah_dict(tr, packer)
+        else:
+            packer = _pack_trace_wout_ah_dict(tr, packer)
+
+    with open(filename, 'wb') as fh:
+        fh.write(packer.get_buffer())
+
+
+def _pack_trace_with_ah_dict(tr, packer):
+
+    # station info
+    packer.pack_int(6)
+    packer.pack_fstring(6, tr.stats.ah.station.code.encode('utf-8'))
+    packer.pack_int(6)
+    try:
+        packer.pack_fstring(6, tr.stats.channel.encode('utf-8'))
+    except:
```

Comment:
code smell.. that try/except was added when there was no encoding yet and it would have never triggered, but due to the encoding being added this is now catching any encoding problems that it's not supposed to catch..

I'll fix it..